### DISCUSSION TITLE
Automation - Bash scripting - timeout?

### DISCUSSION THREAD

**Discussion by upgreydd** [Budibase/budibase#5246] [Help] [answered]
Hey.
I have a question - is there an option to change some timeout for script?
I've added binary which I need to use to `app-service` container and I can confirm existance of binary with:
```
ls -la /path/to/binary
```
binary have `+x` but when I try to execute `/path/to/binary` in automation with `Bash Scripting` I'm getting:
```
{
  "stdout": "spawnSync /bin/sh ETIMEDOUT",
  "success": false
}
```
Any idea why it's happening?

**Comment by upgreydd** [comment, marked as answer]
I've tried to debug it and looks like execution needs to be below 1 second :|

**Reply by melohagan** [reply]
Issue raised here: https://github.com/Budibase/budibase/issues/5248
